.github/ISSUE_TEMPLATE/1-lts-release-checklist.md

```diff
@@ -84,20 +82,23 @@ The [documentation](https://github.com/jenkins-infra/release/blob/master/docs/re
 - [ ] Publish changelog (one day prior to the release in case of a security update).
 
 - [ ] Announce the start of the LTS release process in the [#jenkins-release:matrix.org](https://matrix.to/#/#jenkins-release:matrix.org) channel.
-- [ ] Run job on [release.ci.jenkins.io](https://release.ci.jenkins.io/job/core/job/stable/job/release/) if no security release for Jenkins is planned.
+- [ ] Launch job on [release.ci.jenkins.io](https://release.ci.jenkins.io/job/core/job/stable/job/release/) if no security release for Jenkins is planned.
+  - [ ] Manually review and approve the child release job after carefully checking the "Plan" stage (you can compare with previous stable line).
   - [ ] If this is the first release of a new LTS line, the packaging job will fail on its first run.  Either run the packaging job once and cancel it before the primary release job is run or accept that the packaging job on the first release of a new LTS line will need to be run a second time after it fails the initial run.
+  - [ ] ~3 to 4 hours after the beginning of release job, manually review and approve the child packaging job.
+- [ ] Wait for successful job completion (release: ~3 to 4 hours, packaging ~30 minutes).
 
 - [ ] Check [LTS changelog](https://www.jenkins.io/changelog-stable/) is visible on the downloads site.
 
 - [ ] Publish [GitHub release](https://github.com/jenkinsci/jenkins/releases) pointing to LTS changelog, [sample](https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.387.1).
 
-- [ ] Confirm [Datadog checks](https://p.datadoghq.com/sb/0Igb9a-e6849e5e019250ef5aaea3589297fe8b) are passing.
+- [ ] Confirm that all Packages are available on the Datadog page.
```

Comment:
I think that we should retain the hyperlink so that the release lead can click the link without resorting to bookmarks or some other storage location.  Most release leads are not part of the Jenkins infrastructure team and would not know where to find the Datadog page.
```suggestion
- [ ] Confirm that all Packages are available on the [Datadog page](https://p.datadoghq.com/sb/0Igb9a-e6849e5e019250ef5aaea3589297fe8b).
```

Reply:
I though that as long as a job do this verification it was not very usefull to keep it on this checklist.

Reply:
I'm not aware of any job that does that verification (as far as I know).  When that report has failed in the past, it was an indication of an undetected build problem or an undetected infrastructure problem.  I believe that we created that page specifically to check for incomplete uploads of releases.